src/Fusion.Summary.Api/Controllers/DepartmentsController.cs

```diff
@@ -121,8 +122,12 @@ public async Task<IActionResult> PutV1(string sapDepartmentId, [FromBody] PutDep
         if (string.IsNullOrWhiteSpace(sapDepartmentId))
             return BadRequest("SapDepartmentId route parameter is required");
 
-        if (await ResolvePersonAsync(request.ResourceOwnerAzureUniqueId) is null)
-            return BadRequest("Resource owner not found in azure ad");
+        var personIdentifiers = request.ResourceOwnersAzureUniqueId
+            .Concat(request.DelegateResourceOwnersAzureUniqueId)
+            .Select(p => new PersonIdentifier(p));
+
+        if ((await ResolvePersonsAsync(personIdentifiers)).FirstOrDefault(r => !r.Success) is { } unresolvedProfile)
```

Comment:
Could you rewrite this, a different syntax might make it a bit more readable and help with the understanding

Reply:
Something like this?
```suggestion
             var unresolvedProfiles = (await ResolvePersonsAsync(personIdentifiers))
            .Where(r => !r.Success)
            .ToList();

        if (unresolvedProfiles.Count != 0)
            return BadRequest($"Profiles: {string.Join(',', unresolvedProfiles)} could not be resolved");
```

Reply:
Rewrote it